task_sdk/pyproject.toml

```diff
@@ -114,11 +104,19 @@ codegen = [
     "rich",
     "ruff",
 ]
-
+dev = [
+    "apache-airflow",
```

Comment:
Note that `task_sdk` currently depends on those providers to be present (without them task_sdk tests will not import). We will have to do follow-up steps and cleanup those eventually - but for now they are needed to make the isolated `uv runs to work` - those are also great indicators that there is a cleanup to be done still.